cnf-certification-test/identifiers/identifiers.go

```diff
@@ -663,6 +669,15 @@ var Catalog = map[claim.Identifier]TestCaseDescription{
 		BestPracticeReference: bestPracticeDocV1dot3URL + " Section 5.3.7",
 		ExceptionProcess:      NoDocumentedProcess,
 	},
+	TestContainerIsCertifiedDigestIdentifier: {
+		Identifier:  TestContainerIsCertifiedDigestIdentifier,
+		Type:        normativeResult,
+		Remediation: ContainerIsCertifiedRemediation,
+		Description: formDescription(TestContainerIsCertifiedDigestIdentifier,
+			`Tests whether container images listed in the configuration file have passed the Red Hat Container Certification Program by there digest(CCP).`),
+		BestPracticeReference: bestPracticeDocV1dot3URL + " Section 5.3.7",
```

Comment:
```suggestion
		BestPracticeReference: bestPracticeDocV1dot4URL + " Section 5.3.7",
```

I think the document needs to be the 1.4 doc.